core/cmd.py

```diff
@@ -258,31 +260,24 @@ async def sync_time(self, argument):
             )
         )
 
-    async def rotate180degrees(self, argument):
-        """rotate device 180 degrees"""
-        self.logging.info("starting to rotate device")
-        if argument.lower() == "true":
-            await self.bluetooth.send(Common().rotate180degrees(1))
-        else:
-            await self.bluetooth.send(Common().rotate180degrees(0))
-
-    async def togglescreen(self):
+    async def flip_screen(self, argument: str) -> None:
+        """flip device screen 180 degrees"""
+        self.logging.info("flipping screen")
+        await self.bluetooth.send(Common().flip_screen(argument.upper() == "ON"))
+    
+    async def toggle_screen(self) -> None:
         """toggles the screen on or off"""
         self.logging.info("toggling screen")
-        await self.bluetooth.send(Common().toggleScreenFreeze())
+        await self.bluetooth.send(Common().toggle_screen())
 
-    async def set_brightness(self, argument: str) -> None:
+    async def set_brightness(self, argument: int) -> None:
         """sets the brightness of the screen"""
-        try:
-            conv_brightness = int(argument)
-            if conv_brightness in range (5, 101):
-                self.logging.info(f"setting brightness of the screen: {argument}%")
-                await self.bluetooth.send(Common().set_screen_brightness(brightness_percent=conv_brightness))
-            else:
-                self.logging.error("brightness out of range")
-        except ValueError:
-            self.logging.error(f"Invalid integer: {argument}")
-                    
+        if argument in range (5, 101):
+            self.logging.info(f"setting brightness of the screen: {argument}%")
+            await self.bluetooth.send(Common().set_screen_brightness(argument))
+        else:
+            self.logging.error("brightness out of range")
```

Comment:
I am a fan of giving the user an idea what the range is and what value they provided, just to simplify typing issues. Due to the wide range of users which maybe not technical enough to really know what they are typing in.

Do you agree?

Something like:
`brightness out of range (should be between 5 and 100, given X)`